cpu/x86/x86_pci.c

```diff
@@ -218,7 +218,7 @@ static void pci_find_function(unsigned bus, unsigned dev, unsigned fun)
     printf("  %02x:%02x.%x \"%s\": \"%s\" (%s: %s, rev: %02hhx)\n",
            bus, dev, fun, vendor_name, device_name, baseclass_name, subclass_name, class.revision_id);
 
-    /* cppcheck-suppress memleakOnRealloc */
+    /* c1ppcheck-suppress memleakOnRealloc TODO */
```

Comment:
I used that for disabling the suppressions and checking whether cppcheck actually complained and forgot to take it out. But since its one of the TODOs it doesn't matter that much :) Will fix once I have a good `reason` to add!